### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by obstel**
Translated all labels so far avalaible, a few personal quests are missing.

### CODE REVIEW

**Comment by Lurkars** [comment]
Are those free translation or directly from Feuerland version?

**Comment by obstel** [comment]
Yes, with a little adjustments on my end :)

File: data/fh/label/de.json [hunk 1]
```diff
@@ -535,204 +535,259 @@
...
+        "1": "Erschaffe gefährliches 1-Hex-Gelände in allen an das Ziel angrenzenden,<br> freie Hex-Feldern. Feinde in diesen Hexfeldern leiden darunter",
+        "2": "Erschaffe 1 Hexfeld eisiges Gelände in allen freien Hex-Feldern,<br> die an die Angriffsziele angrenzen"
       },
       "vestige-of-the-imprisoned-god": {
-        "1": "Modify terrain.",
-        "2": "All characters who did not attack suffer trap damage at end of round."
+        "1": "Gelände ändern.",
+        "2": "Alle Charaktere, die nicht angegriffen haben, erleiden am Ende der Runde Fallenschaden."
+      }
+	  "banner-spear": {
```
Comment (Lurkars) [inline data/fh/label/de.json:736]: Not sure how the check passed, may have to adjust. But here is a missing `,`. Please check the files beforehand. In meantime I check how I can test for this.

**Comment by Lurkars** [comment]
Okay, I think the check is working now hopefully. So committing should get rejected on invalid JSON now.

File: data/fh/label/de.json (hunk 1 above)
Comment (obstel) [reply data/fh/label/de.json:736]: Okay, checked it but missed it.

**Pull request merged**